presto-main/src/main/java/com/facebook/presto/operator/PageBufferClient.java

```diff
@@ -196,7 +168,8 @@ else if (completed) {
         }
         String httpRequestState = "not scheduled";
         if (future != null) {
-            httpRequestState = future.getState();
+            // test only
+            httpRequestState = "PROCESSING_REQUEST";
```

Comment:
Should we actually change state to enum? If not, let's at least make sure they follow the same convention. Having both `not scheduled` and `PROCESSING_REQUEST` is a bit confusing.